packages/astro/src/core/errors/errors-data.ts

```diff
@@ -611,6 +611,25 @@ export const PrerenderDynamicEndpointPathCollide = {
 	hint: (filename: string) =>
 		`Rename \`${filename}\` to \`${filename.replace(/\.(?:js|ts)/, (m) => `.json` + m)}\``,
 } satisfies ErrorData;
+
+/**
+ * @docs
+ * @see
+ * - [`getStaticPaths()`](https://docs.astro.build/en/reference/routing-reference/#getstaticpaths)
```

Comment:
It probably feels like knowing about route priority order is the most helpful link here?

https://docs.astro.build/en/guides/routing/#route-priority-order